Dockerfile

```diff
@@ -11,6 +11,11 @@ COPY --chown=sbl:sbl poetry.lock pyproject.toml alembic.ini README.md ./
 RUN poetry config virtualenvs.create false
 RUN poetry install --only main --no-root
 
+ARG IS_MAC
+RUN if [ "$IS_MAC" = "true" ]; then \
+        poetry add polars-lts-cpu; \
```

Comment:
What if we just always install this dependency whether Mac or not? _Seems_ like we could just always have it installed, but then just flip the `LTS_CPU=1` envvar when we want to use it?

If we add it this way, we'll lose dependabot updates, etc. for this package, yes?

Reply:
Ah. Reading up a bit, it seems like you're supposed to use one or the other (`polars` _or_ `polars-lts-cpu`)? I see a number of issues out there where people are grumbling about this package:

- [unionai-oss/pandera#1777](https://github.com/unionai-oss/pandera/issues/1777)
- [pola-rs/polars#12880](https://github.com/pola-rs/polars/issues/12880)
- https://stackoverflow.com/questions/74856814/how-to-import-polars-lts-cpu-correctly

I think what you have he is OK. In general I don't like build args and Docker images that have the potential to work differently locally vs. deployed, but with the current state of that package, I don't think there's much we can do. I think if you just add a comment in there about why we're doing this, and any gotchas about this package, 👍. 

I guess my only question is, is it OK that we're going to have both `polars` _and_ `polars-lts-cpu` installed at the same time? Sounds like they share the same module names, etc. Any chance of non-deterministic behavior?

One solution I saw suggested making both packages optional in Poetry, and then installing one of the other at install time. 🤷

Reply:
I like the optional idea, let me dig into that.  Still would need the env var/build arg (and I'm in agreement with you but don't see a way around it currently), but this would make it one or the other.  When I was originally having to do this regularly when pulling in polars to the filing-api, l didn't see any issues with our filing flow, but I think I was also swapping in the data-validator and pushing that to a branch and pointing filing at it.  So I'll need to update that data-validator to be optional with the libs, and use this flag to determine which one it should use... Starts to get a little messy.

Locally I tested submitting and getting results and all that with both installed and didn't see any issues.

Reply:
>I like the optional idea, let me dig into that.

👍 

>Starts to get a little messy.

Yeah, this whole setup is really not great for packages with Polars as a dependency. The mess just trickles up (down? 🤔😆) the dependency chain.